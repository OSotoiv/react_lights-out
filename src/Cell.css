.Cell {
  height: 100px;
  width: 100px;
  background-color: rgb(0, 156, 109);
  cursor: pointer;
}

.Cell-lit {
  background-color: rgb(139, 11, 11);
}