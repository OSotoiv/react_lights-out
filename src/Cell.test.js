import { render, screen } from '@testing-library/react';
import Cell from './Cell';

test('renders', () => {
    <table>
        <thead></thead>
        <tbody>
            <tr>
                render(<Cell />);
            </tr>
        </tbody>
    </table>
});

test('that is matches snapshot', () => {
    const { asFragment } = render(<Cell />);
    console.log(asFragment)
    expect(asFragment()).toMatchSnapshot();
    //nothing really to getby since the td is empty.
});

