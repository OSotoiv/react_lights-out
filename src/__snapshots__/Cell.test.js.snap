// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`that is matches snapshot 1`] = `<DocumentFragment />`;
